<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScShot</class>
 <widget class="QWidget" name="ScShot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScShot</string>
  </property>
  <widget class="QPushButton" name="push_button_select_socket_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>81</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>state:
ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_select_socket_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>81</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>state:
ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_select_socket_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>159</y>
     <width>81</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>state:
ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_sent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>288</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>SEND=&gt;
(ALT+B)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_lock_all">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>219</y>
     <width>81</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Lock
Buttons</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>16</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_port_1">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>40</y>
     <width>69</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>7011</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_ipaddr_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>202</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>155.94.228.8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>16</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_port_2">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>100</y>
     <width>69</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>7012</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_ipaddr_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>202</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>155.94.228.8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>16</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_port_3">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>160</y>
     <width>69</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>7013</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_remote_ipaddr_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>202</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>155.94.228.8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_startFrpAndListen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_local_ipaddr">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>202</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_local_port">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>0</y>
     <width>69</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>8000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>16</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineEdit_log">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;
background-color: rgb(181, 181, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Click 'START' to run frpc and start a tcp server.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
